### PR TITLE
Met all requirements

### PR DESCRIPTION
**Pull request by Likh-Alex**
- Created models `Order`
- Created DAO `OrderDao`
- Created service `OrderService`

### CODE REVIEW

**Review by vlykhvar — APPROVED** [review]

File: src/main/java/com/dev/cinema/Main.java [hunk 1]
```diff
@@ -109,7 +110,8 @@ public static void main(String[] args) {
             User userByLogin = authenticationService.login(tolik.getEmail(), password);
             System.out.println(userByLogin.toString());
         } catch (AuthenticationException e) {
-            System.out.println("An error encountered while login the user");;
+            System.out.println("An error encountered while login the user");
+            ;
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/Main.java]:
```suggestion
```

File: src/main/java/com/dev/cinema/Main.java [hunk 2]
```diff
@@ -167,5 +170,51 @@ public static void main(String[] args) {
...
+            ticket.setMovieSession(movieSessionForToday);
+        }
+        for (Ticket ticket : Arrays.asList(additionalTicket4, additionalTicket5,
+                additionalTicket6)) {
+            ticket.setMovieSession(movieSessionForTomorrow);
+        }
+        for (Ticket ticket4 : Arrays.asList(additionalTicket1, additionalTicket2,
+                additionalTicket3)) {
+            ticket4.setUser(registeredMykola);
+        }
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/Main.java]: what is the purpose of this?

File: src/main/java/com/dev/cinema/Main.java [hunk 3]
```diff
@@ -167,5 +170,51 @@ public static void main(String[] args) {
...
+        shoppingCartService.addSession(movieSessionForToday, registeredMykola);
+        ShoppingCart shoppingCart4 = shoppingCartService.getByUser(registeredMykola);
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/Main.java]: let's have informative variable names

File: src/main/java/com/dev/cinema/Main.java [hunk 4]
```diff
@@ -167,5 +170,51 @@ public static void main(String[] args) {
...
+        System.out.println("COMPLETE FIRST ORDER FOR SHOPPING CART 4\n");
+        System.out.println(orderService.completeOrder(shoppingCart4));
+        System.out.println("COMPLETED ORDER 4\n");
+        System.out.println(shoppingCart4);
+
+        for (Ticket ticket : Arrays.asList(additionalTicket4, additionalTicket5,
+                additionalTicket6)) {
+            ticket.setUser(registeredMykola);
+        }
```
Comment (KseniiaMakarova) [inline src/main/java/com/dev/cinema/Main.java]: let's remove this?

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/dev/cinema/Main.java (hunk 4 above)
Comment (Likh-Alex) [reply src/main/java/com/dev/cinema/Main.java]: Absolutely agree, removed

File: src/main/java/com/dev/cinema/Main.java (hunk 2 above)
Comment (Likh-Alex) [reply src/main/java/com/dev/cinema/Main.java]: I don't know)I have been building the ordering flow and it got where I got lost.
Removed, thank you

File: src/main/java/com/dev/cinema/Main.java (hunk 1 above)
Comment (Likh-Alex) [reply src/main/java/com/dev/cinema/Main.java]: Done

File: src/main/java/com/dev/cinema/Main.java (hunk 3 above)
Comment (Likh-Alex) [reply src/main/java/com/dev/cinema/Main.java]: Renamed with more informative name

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**